### PR TITLE
[1LP][RFR] removed unused webui statement from test_delete_infra_object

### PR DESCRIPTION
**Pull request by izapolsk**
{{pytest: cfme/tests/infrastructure/test_delete_infra_object.py}}
1. added accidentally removed yaml statements and updated them
2. fixed tests and various other related issues

### CODE REVIEW

**Comment by izapolsk** [comment]
two tests still fail. One test will be fixed along with datastore widgetastic conversion. I'm already working on it.
Second test will probably be fixed by test owner. I didn't manage to fully fix it withing reasonable time.

File: cfme/infrastructure/resource_pool.py [hunk 1]
```diff
@@ -156,8 +157,8 @@ def delete(self, cancel=True, wait=False):
 
         # flash message only displayed if it was deleted
         if not cancel:
-            view.entities.flash.assert_no_error()
-            view.entities.flash.assert_success_message('Delete initiated for 1 Resource Pool')
+            msg = 'The selected Resource Pools was deleted'
```
Comment (psav) [inline cfme/infrastructure/resource_pool.py]: aiieee......MiQ should learn plurals :)

**Comment by psav** [comment]
Failures explanation please

**Comment by mshriver** [comment]
The test_delete_datastore failure is because the cfme_data yaml is wrong for vsphere55 provider, vsanDatastore from remove_test['datastore'] doesn't exist on the provider.

test_delete_cluster failure is because the cluster didn't show back up in time, or when it did it didn't match the cluster object. Not caused by any changes here, just tangentially related.

@psav I'm comfortable merging if you are, given these 2 test failures.  I'm opening a MR against cfme_data.yaml

**Comment by izapolsk** [comment]
@mshriver, as far as I remember I added NFS-Datastore first time but test couldn't find it. so, I replaced it with vsanDatastore. 
I hope it will work with NFS-Datastore. Thanks for fixing this.

**Pull request merged**